diff --git a/node_modules/ffmpeg-kit-react-native/android/build.gradle b/node_modules/ffmpeg-kit-react-native/android/build.gradle
index 2909280..e1feb22 100644
--- a/node_modules/ffmpeg-kit-react-native/android/build.gradle
+++ b/node_modules/ffmpeg-kit-react-native/android/build.gradle
@@ -125,5 +125,5 @@ repositories {
 
 dependencies {
   api 'com.facebook.react:react-native:+'
-  implementation 'com.arthenica:ffmpeg-kit-' + safePackageName(safeExtGet('ffmpegKitPackage', 'https')) + ':' + safePackageVersion(safeExtGet('ffmpegKitPackage', 'https'))
+  implementation 'io.github.maitrungduc1410:ffmpeg-kit-https:6.0.1'
 }
diff --git a/node_modules/ffmpeg-kit-react-native/android/gradle.properties b/node_modules/ffmpeg-kit-react-native/android/gradle.properties
index a912028..1919b35 100644
--- a/node_modules/ffmpeg-kit-react-native/android/gradle.properties
+++ b/node_modules/ffmpeg-kit-react-native/android/gradle.properties
@@ -1,3 +1,3 @@
 android.useAndroidX=true
-ffmpegKit.android.main.version=6.0-2
-ffmpegKit.android.lts.version=6.0-2
+ffmpegKit.android.main.version=6.0.LTS
+ffmpegKit.android.lts.version=6.0.LTS
